src/foam/nanos/notification/NotificationSetting.js

```diff
@@ -70,6 +62,7 @@ foam.CLASS({
         notification.setUserId(user.getId());
         notification.setBroadcasted(false);
         notification.setGroupId(null);
+        notification.setEmailIsEnabled(false);
```

Comment:
How does this become true when sending an email is desired?

Reply:
Another settings is responsible for sending emails - foam.nanos.notification.EmailSetting.